stl/inc/filesystem

```diff
@@ -4337,6 +4337,20 @@ namespace filesystem {
         return _STD filesystem::copy(_From, _To, copy_options::none);
     }
 } // namespace filesystem
+
+#ifdef __cpp_lib_concepts
+namespace ranges {
+    template <>
+    inline constexpr bool enable_borrowed_range<filesystem::directory_iterator> = true;
+    template <>
+    inline constexpr bool enable_borrowed_range<filesystem::recursive_directory_iterator> = true;
+    template <>
+    inline constexpr bool enable_view<filesystem::directory_iterator> = true;
+    template <>
+    inline constexpr bool enable_view<filesystem::recursive_directory_iterator> = true;
+} // namespace ranges
+#endif // __cpp_lib_concepts
+

```

Comment:
No change requested, noting for the record: These explicit specializations work because the primary templates are defined in `<xutility>`:
https://github.com/microsoft/STL/blob/3c2fd04d441d46ec9d914d9cbb621a3bac96c3a5/stl/inc/xutility#L1972-L1973
https://github.com/microsoft/STL/blob/3c2fd04d441d46ec9d914d9cbb621a3bac96c3a5/stl/inc/xutility#L2696-L2698